### ISSUE TITLE
Problem with spike timing

### ISSUE THREAD

**Issue by NoahDolev** [neural-electrophysiology-tool-team/NET#2]
The spike timings we get are incorrect.
In JRC the number of samples is larger by a factor of ~2 than the number of actual samples, so the spike timings it exports are incorrect by that factor as well.
(We took the correct number of samples from the .h5 file, and the JRC config file has the correct sample rate of 20kHz, so the problem is probably upstream).

the setup file *_layout_100_16x16_newSetup_JRC_ksort.mat contains the variable "rez".
rez.ops.sampsToRead is already larger by this factor, so maybe the problem is in the h5 to binary conversion?

**Comment by corticodes** [comment]
Doesn't seem to be a problem in the bin file or the conversion..
Probably something in the kiloSort config/params file. I'll try to have a look.

Yuval

**Comment by corticodes** [comment]
I just ran getKiloSorting on one of the old recording (made by Mark in the old MC system) and the sampsToRead that was in the rez file matched the binaryRecording's nTotSamples.
I also exported a .h5 recording to binary and ran getKiloSorting, and the numbers again matched.
Make sure that the path in PCspecificFiles\kiloSortOpsPath.txt is right and the config file is relevant to the file you are trying to sort. I don't know if the number of samples written there overrides anything but it is saved there so using the wrong config may cause this problem.
